lib/index.js

```diff
@@ -196,38 +215,110 @@ exports.init = function(options, callback) {
-    return text.join('\n');
-};
+exports.print = function(sorted) {
+    console.log(exports.asTree(sorted));
+};
+
+exports.asTree = function(sorted) {
+    return treeify.asTree(sorted, true);
+};
+
+exports.asCSV = function(sorted, customFormat) {
+    var text = [ ], textArr = [ ], lineArr = [ ];
+
+    if (customFormat && Object.keys(customFormat).length > 0) {
+        textArr = [ ];
+        textArr.push('"module name"');
+
+        Object.keys(customFormat).forEach(function forEachCallback(item) {
+            textArr.push('"' + item + '"');
+        });
+
+        text.push(textArr.join(','));
+    } else {
+        text.push(['"module name"','"license"','"repository"'].join(','));
+    }
+
+    Object.keys(sorted).forEach(function(key) {
+        var module = sorted[key],
+        line = '';
+        lineArr = [ ];
+
+        //Grab the custom keys from the custom format
+        if (customFormat && Object.keys(customFormat).length > 0) {
+            lineArr.push('"' + key + '"');
+            Object.keys(customFormat).forEach(function forEachCallback(item) {
+                lineArr.push('"' + module[item] + '"');
+            });
+            line = lineArr.join(',');
+        } else {
+            line = [
+                '"' + key + '"',
+                '"' + (module.licenses || '') + '"',
+                '"' + (module.repository || '') + '"'
+            ].join(',');
+        }
+        text.push(line);
+    });
+
+    return text.join('\n');
+};
+
+/**
+* Exports data as markdown (*.md) file which has it's own syntax.
+* @method
+* @param  {JSON} sorted       The sorted JSON data from all packages.
+* @param  {JSON} customFormat The custom format with information about the needed keys.
+* @return {String}            The returning plain text.
+*/
+exports.asMarkDown = function(sorted, customFormat) {
+
+    var text = [];
+    if (customFormat && Object.keys(customFormat).length > 0) {
+
+        Object.keys(sorted).forEach(function sortedCallback(sortedItem) {
+
+            text.push(' - **[' + sortedItem + '](' + sorted[sortedItem].repository + ')**');
+            Object.keys(customFormat).forEach(function customCallback(customItem) {
+
+                text.push('    - ' +  customItem + ': ' + sorted[sortedItem][customItem]);
+            });
+        });
+
+        text = text.join('\n');
+    } else {
+
+        Object.keys(sorted).forEach(function(key) {
+
+            var module = sorted[key];
+            text.push('[' + key + '](' + module.repository + ') - ' + module.licenses);
+        });
+
+        text = text.join('\n');
+    }
+
+    return text;
+};
+
+/*istanbul ignore next*/
+exports.parseJson = function(jsonPath) {
+    if (typeof jsonPath !== 'string') {
+        return new Error('did not specify a path');
+    }
+
+    var jsonFileContents = '',
+        result = { };
+
+    try {
+        jsonFileContents = fs.readFileSync(jsonPath, {encoding: 'utf8'});
+        result = JSON.parse(jsonFileContents);

```

Comment:
Mh... I didn't know about that syntax. One line seems to be more elegant. But I'll leave it to you since I'm really unsure about the implementation. :S